arquillian/adapter/src/main/java/org/wildfly/swarm/arquillian/adapter/UberjarSimpleContainer.java

```diff
@@ -387,6 +388,22 @@ private String gav(final String group, final String artifact, final String versi
 
     }
 
+    private void deleteRecursively(File file) {
+        if (!file.exists()) {
+            return;
+        }
+        if (file.isDirectory()) {
+            File[] files = file.listFiles();
+            if (files != null) {
+                for (File child : files) {
+                    deleteRecursively(child);
+                }
+            }
+        }
+
+        file.delete();
```

Comment:
According to what we found out together, I think Files.delete(Path) is more reliable or at least it has better error reporting. We may want to switch to that API here.